### ISSUE TITLE
custom emoji example type error

### ISSUE THREAD

**Issue by Eraldo** [TypeCtrl/ngx-emoji-mart#48]
When using the custom emoji example from the docs:
```
const customEmojis = [
  {
    name: 'Octocat',
    short_names: ['octocat'],
    text: '',
    emoticons: [],
    keywords: ['github'],
    imageUrl: 'https://assets-cdn.github.com/images/icons/emoji/octocat.png?v7'
  },
]
```

I get this trace when opening the picker:
```
Error: Uncaught (in promise): TypeError: Cannot create property 'custom' on string 'octocat'
TypeError: Cannot create property 'custom' on string 'octocat'
    at EmojiService.webpackJsonp.1225.EmojiService.getData (http://localhost:8100/build/49.js:25919:30)
    at http://localhost:8100/build/49.js:341:73
    at Array.filter (<anonymous>)
    at CategoryComponent.webpackJsonp.1224.CategoryComponent.getEmojis (http://localhost:8100/build/49.js:341:22)
    at CategoryComponent.webpackJsonp.1224.CategoryComponent.ngOnInit (http://localhost:8100/build/49.js:287:28)
    at checkAndUpdateDirectiveDynamic (http://localhost:8100/build/vendor.js:12773:19)
    at checkAndUpdateNodeDynamic (http://localhost:8100/build/vendor.js:14290:20)
    at checkAndUpdateNode (http://localhost:8100/build/vendor.js:14214:16)
    at debugCheckAndUpdateNode (http://localhost:8100/build/vendor.js:15104:76)
    at debugCheckDirectivesFn (http://localhost:8100/build/vendor.js:15045:13)
    at c (http://localhost:8100/build/polyfills.js:3:19752)
    at http://localhost:8100/build/polyfills.js:3:19174
    at http://localhost:8100/build/polyfills.js:3:21242
    at t.invoke (http://localhost:8100/build/polyfills.js:3:14976)
    at Object.onInvoke (http://localhost:8100/build/vendor.js:5124:33)
    at t.invoke (http://localhost:8100/build/polyfills.js:3:14916)
    at r.run (http://localhost:8100/build/polyfills.js:3:10143)
    at http://localhost:8100/build/polyfills.js:3:20242
    at t.invokeTask (http://localhost:8100/build/polyfills.js:3:15660)
    at Object.onInvokeTask (http://localhost:8100/build/vendor.js:5115:33)
```

**Comment by scttcper** [comment]
Hey I tried to recreate the issue with no success

https://stackblitz.com/edit/angular-hcbtrj?file=app%2Femoji.component.ts

**Comment by Eraldo** [comment]
True.
I just took a look at your demo.
To be honest, I don't know why it works there.

I took a screenshot of the position where it is happening:
![screen shot 2018-04-17 at 18 29 10](https://user-images.githubusercontent.com/73378/38883516-6c1b01a6-426d-11e8-8d2a-7cf3739567e9.png)

It tries to access a property `custom` but my custom emoji does not have that attribute.
I do not understand why in your example it does not trip over that. *puzzled*.
Any ideas?
